hal/src/photon/lib/BESL.ARM_CM3.release.a.weaken

```diff
@@ -37,3 +37,24 @@ rsa_pkcs1_verify
 rsa_free
 rsa_check_pubkey
 rsa_check_privkey
+mbedtls_ccm_init
+mbedtls_ccm_setkey
+mbedtls_ccm_free
+mbedtls_ccm_encrypt_and_tag
+mbedtls_ccm_auth_decrypt
+mbedtls_cipher_info_from_type
+mbedtls_cipher_info_from_string
+mbedtls_cipher_info_from_values
+mbedtls_cipher_init
```

Comment:
We should definitely verify that WPA Enterprise authentication still works after this change.

Reply:
Added to Steps to test